File: Submissions/T07/H1.py
n = int(input("Masukkan N : "))
data = list(map(int, input("Masukkan data : ").split()))

data.sort()

minimum = min(data)
maximum = max(data)
rata_rata = sum(data) / n

print("Data terurut :", *data)
print(f"Min: {minimum} | Max: {maximum} | Rata-rata: {rata_rata:.2f}")

File: Submissions/T19/H1/h1.c
#include <stdio.h>
int main() {
    int n;
    printf("Masukkan N : "); scanf("%d", &n);
    int data[n];
    printf("Masukkan data : ");
    for (int i = 0; i < n; i++) scanf("%d", &data[i]);
    for (int i = 0; i < n - 1; i++)
        for (int j = 0; j < n - i - 1; j++)
            if (data[j] > data[j+1]) { int tmp = data[j]; data[j] = data[j+1]; data[j+1] = tmp; }
    double sum = 0;
    printf("Data terurut :");
    for (int i = 0; i < n; i++) { printf(" %d", data[i]); sum += data[i]; }
    printf("\nMin: %d | Max: %d | Rata-rata: %.2f\n", data[0], data[n-1], sum / n);
}
